docs/resources/sidecar_listener.md

```diff
@@ -0,0 +1,121 @@
+# cyral_sidecar_listener (Resource)
```

Comment:
Could you please run the `docker-compose/docs` script to update the doc with the latest changes? Thanks